src/app/services/util/util.service.ts

```diff
@@ -38,7 +38,13 @@ export class UtilService {
     let selectedLanguage:any = localStorage.getItem("preferred_language")
     return JSON.parse(selectedLanguage)?.value || "en"
   }
-  isFileType(fileUrl: string, type: 'images' | 'pdf' | 'doc' | 'video'): boolean {
-    return this.regex[type].test(fileUrl);
+
+  isFileType(fileType: string) {
```

Comment:
We have tested for all these file types right? images' | 'pdf' | 'doc' | 'video'